plugin.json

```diff
@@ -90,5 +90,10 @@
             "placeholder": "demo_plugin",
             "default": "demo_plugin"
         }]
+    },
+    "required_configuration" : {
```

Comment:
@isacikgoz you need to update the server dependency to your commit https://github.com/mattermost/mattermost-server/pull/13021/commits/4a04d39d5a3d9e4202547159e312a86d35636374

Reply:
@hanzei I was going to update the PR once [server PR](https://github.com/mattermost/mattermost-server/pull/13021) is merged. For now I just added this to mod file to use my fork:`replace github.com/mattermost/mattermost-server => ../mattermost-server`. Hence, I didn't include mod file in the PR. Should I add something like: `github.com/mattermost/mattermost-server => github.com/isacikgoz/mattermost-server@mycommitid`?

Reply:
I was to fast on my comment. Let's deffer the review of this PR until https://github.com/mattermost/mattermost-server/pull/13021 is merged.

Reply:
Given that the server PR is approved from both dev I think we are save to continue the review on this PR.

@isacikgoz would your please update the dependency to point to your commit?